Parity/src/QwBPMStripline.cc

```diff
@@ -380,12 +395,23 @@ void  QwBPMStripline<T>::ProcessEvent()
   */
 
   fEffectiveCharge.ClearEventData();
+  fEllipticity.ClearEventData();
 
   for(i=0;i<4;i++)
     {
       fWire[i].ProcessEvent();
       fEffectiveCharge+=fWire[i];
+      if (i<2)
+      {
+        fEllipticity+=fWire[i];
+      }
+      else 
+      {
+        fEllipticity-=fWire[i];
+      }
     }
+  fEllipticity.Ratio(fEllipticity,fEffectiveCharge);
```

Comment:
Is there some technique to catch division by zero and turn it into a device error?

Reply:
If we wanted to set a device error for attempting to divide by zero, the place to do it would be in the "operator/=" functions.
Right now, we would just get a warning message (and a data structure set to zero).